pkg/kudoctl/util/resources/install.go

```diff
@@ -0,0 +1,115 @@
+package resources
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+// Install installs package resources.
+// If skipInstance is set to true, only a package's Operator and OperatorVersion is installed.
+func Install(kc *kudo.Client, resources *packages.PackageCRDs, skipInstance bool, instanceName, namespace string, parameters map[string]string) error {
```

Comment:
Why do we need a new package (underneath `util` (!)) that deals with the logic of installing a new operator into the cluster? Most of the called methods are already well placed in the `kudo.Client`. The logic of calling them in the right sequence and doing the error handling is IMHO better placed in the CLI (or some overarching method in `kudo.Client`?)

Reply:
Thinking about this, this could be refactored even more, maybe not in this PR. I think this could really be 3 different functions that splits this massive method up by it's constituent parts, and then the CLI package handles the skipInstance logic.

Reply:
👍

Reply:
>  I think this could really be 3 different functions

agree. I was started to feel uneasy about method size